src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
   // write your code here
+  bite(animal) {
+    if (animal.hidden === false) {
```

Comment:
Also check here instance of animal according task point
![image](https://user-images.githubusercontent.com/70009205/230438688-da22ed55-ee86-4e20-b398-0f26b5140f4c.png)

```suggestion
    if (!animal.hidden) {
```